app/components/qr-code.tsx

```diff
@@ -85,12 +102,14 @@ export function QRCode({
               className="rounded-lg bg-foreground"
             />
           ) : (
-            <QRCodeSVG
-              value={value}
-              size={256}
-              marginSize={3}
-              className="rounded-lg bg-foreground"
-            />
+            <ErrorBoundary fallback={<QRCodeFallback value={value} />}>
```

Comment:
are there any errors other than value too long that could happen and when we maybe wouldn't want to show this fallback?

Reply:
yeah. seems fine